scripts/ci/update-changelog.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash -eux
+
+
+dependencies="cargo git git-cliff"
+
+for dependency in $dependencies
+do
+  if ! command -v $dependency &>/dev/null
+  then
+    echo "error: $dependency was not found in PATH" >&2
+    exit 255
+  fi
+done
+
+# The default "starting" commit is a somewhat arbitrary starting point for
+# cataloging recent commits in a way that breaks from the old convention
+DEFAULT_START_COMMIT="891e7d47827e99947e46c82f509e479aa13acf24"
```

Comment:
Similar question here as to the "initial" commit of the SHA above

Reply:
Same answer, with the difference being that `DEFAULT_START_COMMIT` is not meant to be (regularly) updated, whereas the commit in the text file should get updated every time the automation checks for new changes.